### PR TITLE
updates to Localytics 4.0

### PR DESCRIPTION
**Pull request by ladanazita**
https://docs.localytics.com/dev/android.html#migrating-v3-to-v4-android

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ dependencies {
 
   provided 'com.segment.analytics.android:analytics:4.0.0'
 
-  compile 'com.localytics.android:library:3.8.1'
+  compile 'com.localytics.android:library:4.0+'
```
Comment (f2prateek) [inline build.gradle:49]: Convention is to to use `4.0.+`

Comment (ladanazita) [inline build.gradle:49]: Is this OK now?

Comment (f2prateek) [inline build.gradle:49]: yup

**Comment by f2prateek** [comment]
I changed this to 4.0.1 because maven doesn't like + in the version https://cloudup.com/cyYZFQtwCVw

**Pull request merged**